compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -543,6 +545,7 @@ populateConfigInfo(const llvm::SetVector<linalg::LinalgOp> &computeOps,
           return failure();
         }
         setLoweringConfig(linalgOp, *loweringConfig);
+        break;
```

Comment:
what's happening here?

Reply:
Making sure that once we attach the lowering config, we can just bail out. Since we are iterating over the operand in the inner most loop.

Reply:
Can you add a comment here?